.travis.yml

```diff
@@ -45,6 +60,7 @@ addons:
   apt:
     packages:
       - python-dev
+#      - python2-dev
```

Comment:
Now python2 support is going to be broken on all 3 Ubuntu versions. We need separate `apt` sections for 20.04 vs. the others, if that's possible.

Python2 support **is** possible on Ubuntu 20.04, but only if you specifically install the 1.67.0 versions of the Boost libraries. Which is what I have done.

Reply:
...Or maybe python2 support won't be broken on Xenial and Bionic. Never mind that part then.

I would still like to get py2 working on Travis Focal though. Perhaps this would be a good time to implement #176 , and include a `bootstrap` script that installs any necessary development dependencies on that specific distro. I'm working on that right now, actually.

Reply:
What's the bottom line here though? can we merge this?

Reply:
Let's make a decision about https://github.com/vegastrike/Vega-Strike-Engine-Source/pull/284/files#r471870159 and then this also, and we can decide.

1. We can't break Py2 support.
2. Question: do we want to start requiring Py3 builds to pass on master? (We need to make sure they're not required for 0.6.x.)

Depending on our answer to '2' we might make a decision specifically about Focal too for practical purposes as far as our provided packages go.

Reply:
Py3 shows to be very stable (both in buildtime, and runtime according to @evertvorster), and since we do won't to convert to it, I think we should start requiring it as to make sure no one breaks what we already have.
If there is some doubts about this, we can still add this to the allowed failures list, though I think that would be wrong considering where we are standing now

Reply:
@nabaco I agree. I have also switched to using Py3 by default. There are a few runtime errors, but the game seems to work fine. And it compiles very reliably.

Reply:
waiting for your approval to get this merged :)